kuka_eki_hw_interface/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ find_package(catkin REQUIRED COMPONENTS
   hardware_interface
   joint_limits_interface
   roscpp
+  angles
```

Comment:
Could you move `angles` to the top? Let's keep these `COMPONENTS` sorted alphabetically.